packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -32,24 +32,28 @@ enum Orientation {
 /// exception, unless the `nullOk` argument is set to true, in which case it
 /// returns null.
 ///
-/// MediaQueryData includes two [EdgeInsets] values:
-/// [padding] and [viewInsets]. These
-/// values reflect the configuration of the device and are used by
-/// many top level widgets, like [SafeArea] and the Cupertino and
-/// Material scaffold widgets. The padding value defines areas that
-/// might not be completely visible, like the display "notch" on the
-/// iPhone X. The viewInsets value defines areas that aren't visible at
-/// all, typically because they're obscured by the device's keyboard.
+/// MediaQueryData includes three [EdgeInsets] values:
+/// [padding], [viewPadding], and [viewInsets]. These values reflect the
+/// configuration of the device and are used by many top level widgets, like
+/// [SafeArea] and the Cupertino and Material [Scaffold] widgets. The padding
```

Comment:
nit: maybe instead of linking to [Scaffold] only, which is the material, we could link to all of them somehow.  Perhaps as a see also, or perhaps in-line here?